### ISSUE TITLE
Canadian EV - Sometimes status return undefined

### ISSUE THREAD

**Issue by guyaumetremblay** [SondreNjaastad/node-red-contrib-bluelinky#21]
Hi, 

Sometimes, I got `status: undefined` from the `car status` node. Sometimes, calling it a second time work. 
<img width="189" alt="Capture d’écran, le 2021-01-17 à 16 34 19" src="https://user-images.githubusercontent.com/17008947/104856642-daff9180-58e1-11eb-8a66-ee73661defa7.png">

I use the not parsed version, with `do refresh = true`. 

I have a Canadian Kona EV. 

Not sure how to provide more informations to debug this. Let me know... 
Also, let me know if this should be open in https://github.com/Hacksore/bluelinky instead.

Thanks!

**Comment by Hacksore** [comment]
I feel like I've seen this before in the `US` region and it might have to do with refreshing the token.

Will try to pull status again after 30 minutes to see if I get `undefined` as well and provide an update here.

**Comment by Hacksore** [comment]
Was able to reproduce after waiting 30 minutes so something is up, will take a look and see if I can find the root cause.

![image](https://user-images.githubusercontent.com/996134/105068989-136cb000-5a47-11eb-9b32-a7e4a46c6d1f.png)

**Comment by Hacksore** [comment]
So I had a look and this seems to be the fix for `US` vehicles, I'll look over the code for `CA` and see if I can apply the same thing. In short, this is going to require a bluelinky update to fix the issue.

https://github.com/Hacksore/bluelinky/commit/8ca38afc7758aeebf27de5acaafb6ecc650154ca

**Comment by guyaumetremblay** [comment]
@Hacksore yep seems to me that there is the same error in the calculation of the variable "shouldRefreshToken" @ https://github.com/Hacksore/bluelinky/blob/558af06b0fbd364201202abbfc4f47066a0ca192/lib/controllers/canadian.controller.ts#L22

European controller has the same version than the american one.

Tell me if you need someone to test the fixed version.

**Comment by Hacksore** [comment]
@guyaumetremblay just committed the working of `v6.0.0-alpha1` to the bluelinky repo and there is an open PR for it to be released. 

Once that is merged/published I can try updated the node-red repo and see if that fixes this issue.

**Comment by guyaumetremblay** [comment]
@Hacksore Hi, I'm sorry but I still received undefined with the new version. Here is what I did in my flow:
- call Door unlock - > received "Unlock sucessful" 
- wait 5sec
- call Get status - > received "status: undefined"

Calling "Get status" only, after or before, worked! (see the debug screenshot) 

![Screenshot_20210124-145027_Home Assistant](https://user-images.githubusercontent.com/17008947/105641717-9d57b700-5e53-11eb-9824-66abe328a76a.jpg)
![Screenshot_20210124-145421_Home Assistant](https://user-images.githubusercontent.com/17008947/105641849-2bcc3880-5e54-11eb-8ad8-e171febc8814.jpg)

**Comment by Hacksore** [comment]
@guyaumetremblay ok let me test again and see what I find :)

**Comment by Hacksore** [comment]
Ok so I don't quite this is a token issue anymore as the tokens in CA last a whole day from what I found. @guyaumetremblay can we track this in a separate issue if you don't mind. 

```log
"expireIn": 86400
```

**Comment by guyaumetremblay** [comment]
@Hacksore Ok I opened #23
